test_tlspuffin.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Is this file used anywhere?

Reply:
Nope. I was apparently too eager to add files in my commits. This and `test_mk_vendor.sh` are local scripts of mine that should be removed. I'll revert that.